pkgs/os-specific/linux/ati-drivers/fglrx_gpl_symbol.patch

```diff
@@ -0,0 +1,11 @@
+--- 13.1/common/lib/modules/fglrx/build_mod/firegl_public.c	2013-01-15 22:33:27.000000000 +0100
++++ 13.1/common/lib/modules/fglrx/build_mod/firegl_public.c	2014-08-18 20:05:44.327520475 +0200
+@@ -250,7 +250,7 @@
+ #endif
+ 
+ #ifdef MODULE_LICENSE
+-MODULE_LICENSE("Proprietary. (C) 2002 - ATI Technologies, Starnberg, GERMANY");
++MODULE_LICENSE("GPL\0Proprietary. (C) 2002 - ATI Technologies, Starnberg, GERMANY");
```

Comment:
I think I know how to solve this but I won't be re-submitting this merge.
I don't use this version of the software and back ported it as a courtesy.
I have absolutely no interest in maintaining this software either, considering the maintainers are no where to be found and I have spent the last month debugging and testing this drv so that I can use nixos without having to put up with the noisy open source radeon driver (power management support is bad. Performance even worse.)
So I will come back with a fix but I will not be applying it.

Reject the merge if you want. I couldn't care less. I am perfectly happy to exercise my right to .gitignore